generators/setup-docker/configTransformers.js

```diff
@@ -156,8 +160,11 @@ function transformChannelConfig(channelJsonFormat, orgsTransformed) {
 
   return {
     key: channelJsonFormat.key,
-    name: channelJsonFormat.name,
+    name: channelName,
     orgs: orgsForChannel,
+    profile: {
+      name: _.chain(channelName).camelCase().upperFirst().value(),
```

Comment:
I would create a separate variable for that, for example `const profileName` just after `const channelName`